modules/extract/functions/EXTRACT/fn_EXTRACT_CheckSideInVeh.sqf

```diff
@@ -0,0 +1,12 @@
+#include "script_component.hpp"
+
+private _veh = _this select 0;
+private _side = _this select 1;
+
+private _units = [];
+
+{if (side _x == _side) then {_units set [count _units, _x]}} forEach playableUnits;
```

Comment:
No need to set array, you can just use the playableUnits array and select elements from it

```sqf
private _units = playableUnits select {side _x isEqualTo _side};
```